tools/data-handler/src/show.ts

```diff
@@ -74,6 +78,89 @@ export class Show {
     if (!cardKey) {
       throw new Error(`Mandatory parameter 'cardKey' missing`);
     }
+
+    const attachment = await this.getAttachment(projectPath, cardKey, filename);
+
+    let attachmentPath: string = '';
+    if (attachment) {
+      attachmentPath = `${attachment.path}/${attachment.fileName}`;
+    }
+
+    if (!attachment || !existsSync(attachmentPath)) {
+      throw new Error(`Attachment '${filename}' not found for card ${cardKey}`);
+    } else {
+      const fileBuffer = readFileSync(attachmentPath);
+      let mimeType = mime.lookup(attachmentPath);
+      if (mimeType === false) {
+        mimeType = 'application/octet-stream';
+      }
+      const payload: attachmentPayload = { fileBuffer, mimeType };
+      return payload;
+    }
+  }
+
+  /**
+   * Opens an attachment using a configured application or the operating system's default application.
+   * @param key cardkey of the attachment
+   * @param filename attachment filename
+   * @param waitDelay amount of time to wait for the application to open the attachment
+   */
+  public async openAttachment(
+    projectPath: string,
+    cardKey: string,
+    filename: string,
+    waitDelay: number = 1000,
+  ) {
+    const attachment = await this.getAttachment(projectPath, cardKey, filename);
```

Comment:
I think we should at some stage move the `getAttachment` function to somewhere else, it is not that much related to `Show`.  And then move the `openAttachment` functionality also somewhere else, but let's keep them here for now.

Reply:
Yes I had the same thoughts but I figured the CLI command would be something like `cyberismo show attachment xxx file.png`